programmers/체육복.js

```diff
@@ -0,0 +1,16 @@
+function solution(n, lost, reserve) {
+    let realLost = lost.filter(i => reserve.indexOf(i) === -1).sort((a, b) => a - b);
+    let realReserve = reserve.filter(i => lost.indexOf(i) === -1).sort((a, b) => a - b);
```

Comment:
배열의 변수명은 무조건 복수로 선언해주세요.
```solution``` 함수의 매개변수들도 복수형으로 바꿔주었습니다.
각 배열의 ```filter```를 돌릴 때 내부 원소들을 **단수**로 지어줬습니다.

또한 내부 원소의 존재여부를 판단할 때는
```indexOf()``` 가 아닌 ```includes()``` 를 사용해주세요.
이름만 봐도 명확히 의미를 파악할 수 있으니까요.
기억하셔야 합니다.
절대 까먹지 말길!!

```suggestion
function solution(n, losts, reserves) {
    let realLost = losts.filter(lost => reserves.includes(lost)).sort((a, b) => a - b);
    let realReserve = reserves.filter(reserve => losts.includes(reserve)).sort((a, b) => a - b);
```